foxpuppet/windows/__init__.py

```diff
@@ -3,5 +3,5 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from foxpuppet.windows.base import BaseWindow  # noqa: F401
-from foxpuppet.windows.browser import BrowserWindow  # noqa: F401
+from foxpuppet.windows.browser.window import BrowserWindow  # noqa: F401
```

Comment:
This is the `__init__.py` file, so we can use `from foxpuppet.windows import BrowserWindow`. Did you mean something else?